.github/workflows/ipc_fuzzer.yml

```diff
@@ -6,7 +6,21 @@ name: IPC fuzzing
 # 'Actions' tab
 
 # yamllint disable-line rule:truthy
-on: [pull_request, workflow_dispatch]
+on:
+  workflow_call:
+    inputs:
+      fuzzing_duration_s:
+        type: number
+        default: 300  # 5 minutes
```

Comment:
This PR is only about GitHub Actions, nothing about Jenkins or Intel CI

Here's such a 30 minutes run (and everything else) in my fork:
https://github.com/marc-hb/sof/actions/runs/7147973857/job/19468358070

Reply:
They're run on the same runners as Pull Requests, so 30 minutes fuzzing is equivalent to 6 pull requests per day (all the rest is equivalent to 1 PR/day). All this can of course be adjusted with one-line change.

@andyross any recommendation about the fuzzing duration?